drivers/usb/device/usb_dc_stm32.c

```diff
@@ -472,24 +471,14 @@ static int usb_dc_stm32_init(void)
 	HAL_StatusTypeDef status;
 	int ret;
 	unsigned int i;
-
```

Comment:
It would be better the empty line is preserved (separates local variable definitions from instructions).